v-next/hardhat-keystore/test/helpers/setup-keystore-file-location-override-at.ts

```diff
@@ -0,0 +1,34 @@
+import type { HardhatPlugin } from "@ignored/hardhat-vnext/types/plugins";
+
+export const setupKeystoreFileLocationOverrideAt = (
+  keystoreFilePath: string,
+): HardhatPlugin => {
+  const hardhatKeystoreFileLocationOverridePlugin: HardhatPlugin = {
+    id: "hardhat-keystore-file-location-override",
+    hookHandlers: {
```

Comment:
This inline hookhandler triggers a warning about not having inline in production.

When we shifted it to dynamic imports we ran into the problem of passing in state (i.e. `keystoreFilePath`) to the hook handler.

Reply:
Let's ignore those warnings for now